app/controllers/projects_controller.rb

```diff
@@ -75,7 +87,7 @@ def filters
         @projects = @projects.independent_projects
       end
     end
-    @pagy, @projects = pagy(@projects.order(created_at: :desc), items: Company::PAGE_SIZE)
+    load_pagy
```

Comment:
Why not add it to before_action too?

Reply:
Becoz @project k and data change ho raha ha